### PR TITLE
doc: update description text file

### PR DESCRIPTION
**Pull request by aakritisparkeighteen**
Update description.txt file

### CODE REVIEW

File: description.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-This is a description file for git training session, now resolve conflicts 
\ No newline at end of file
+This is a description file for git training session, now resolve conflicts 
+
+Learning and making Changes in Description file.
```
Comment (raghav-gkmit) [inline description.txt:3]: @aakritigkmit You have to add a description of the things you have learnt in yesterday's session.

**Comment by aakritisparkeighteen** [comment]
Can you see it again

**Comment by aakritisparkeighteen** [comment]
Hi sir
I wanted to inform you that I’m facing a family emergency and will need to
take leave tomorrow, 2025-04-25 I’ll ensure any pending work is updated
today and can provide handover details if needed.

Thank you for your understanding.
Best regards


*Aakriti Khanna*Development Intern

[image: Photo] <https://www.gkmit.co/>
E:  ***@***.***
P:  +917827634288
A:  A-SQUARE, 2nd floor, 100 Feet Rd, Shobhagpura
<https://goo.gl/maps/YLPZ1TGAXTL72J3S6>, Udaipur, Rajasthan 313011
<https://goo.gl/maps/YLPZ1TGAXTL72J3S6>

[image: facebook icon] <https://www.facebook.com/gkmit.co>  [image: twitter
icon] <https://twitter.com/i/flow/login?redirect_after_login=%2FGKMIT1>
 [image:
linkedin icon] <https://www.linkedin.com/company/gkmit-pvt-ltd/>

Disclaimer: This email and its attachments may contain privileged and
confidential information. If you are not an intended recipient you are
requested not to reproduce, copy, disseminate or in any manner distribute
this email communication as the same is strictly prohibited. If you have
received this email in error, please notify the sender immediately by
return e-mail and delete the communication sent in error. Email
communications cannot be guaranteed to be secure and error-free and GKM IT
Pvt. Ltd. is not liable for any errors in the email communication or for
the proper, timely and complete transmission thereof.

**Comment by aakritisparkeighteen** [comment]
Noted.

Aayush Sharma
Full Stack Lead
***@***.***
P: +91-7665352012
A:  A-SQUARE, 2nd floor, 100 Feet Rd, Shobhagpura, Udaipur, Rajasthan 313011

Disclaimer: This email and its attachments may contain privileged and confidential information. If you are not an intended recipient you are requested not to reproduce, copy, disseminate or in any manner distribute this email communication as the same is strictly prohibited. If you have received this email in error, please notify the sender immediately by return e-mail and delete the communication sent in error. Email communications cannot be guaranteed to be secure and error-free and GKM IT Pvt. Ltd. is not liable for any errors in the email communication or for the proper, timely and complete transmission thereof.

**Comment by aakritisparkeighteen** [comment]
above files are attached for report generation .


On Mon, Sep 29, 2025 at 1:19 PM Karthik Arumugam ***@***.***> wrote:

>
>
> ---------- Forwarded message ---------
> From: ***@***.***>
> Date: Tue, 16 Sept 2025 at 16:42
> Subject: Formats of eXlr8 Dashboard report
> To: Sreenivas Makam ***@***.***>, Karthik Arumugam <
> ***@***.***>
> Cc: Shishir Sharma ***@***.***>, Russell Araujo ***@***.***>,
> Himanshu Vashistha ***@***.***>
>
>
> Hi Team,
>
>
>
> Pls find the attached formats of MIS, Stock Summary and Wallet Transaction
> reports for your reference.
>
>
>
> Thanks & Regards,
>
> KGen-Sales
>
>
>


-- 

*Aakriti Khanna*Jr. Backend Engineer

[image: Photo] <https://www.gkmit.co/>
E:  ***@***.***
P:  +917827634288
A:  A-SQUARE, 2nd floor, 100 Feet Rd, Shobhagpura
<https://goo.gl/maps/YLPZ1TGAXTL72J3S6>, Udaipur, Rajasthan 313011
<https://goo.gl/maps/YLPZ1TGAXTL72J3S6>

[image: facebook icon] <https://www.facebook.com/gkmit.co>  [image: twitter
icon] <https://twitter.com/i/flow/login?redirect_after_login=%2FGKMIT1>
 [image:
linkedin icon] <https://www.linkedin.com/company/gkmit-pvt-ltd/>

Disclaimer: This email and its attachments may contain privileged and
confidential information. If you are not an intended recipient you are
requested not to reproduce, copy, disseminate or in any manner distribute
this email communication as the same is strictly prohibited. If you have
received this email in error, please notify the sender immediately by
return e-mail and delete the communication sent in error. Email
communications cannot be guaranteed to be secure and error-free and GKM IT
Pvt. Ltd. is not liable for any errors in the email communication or for
the proper, timely and complete transmission thereof.

**Pull request closed without merging**